frontends/web/src/routes/account/summary/balancerow.tsx

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright 2023 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useTranslation } from 'react-i18next';
+import { AccountCode, CoinCode, IBalance } from '../../../api/account';
+import { route } from '../../../utils/route';
+import Logo from '../../../components/icon/logo';
+import { Amount } from '../../../components/amount/amount';
+import Spinner from '../../../components/spinner/ascii';
+import { FiatConversion } from '../../../components/rates/rates';
+import style from './accountssummary.module.css';
+import { useEffect, useRef, useState } from 'react';
+import { syncAddressesCount } from '../../../api/accountsync';
+import { TUnsubscribe } from '../../../utils/transport-common';
+
+type TProps = {
+  code: AccountCode;
+  name: string;
+  coinCode: CoinCode;
+  balance?: IBalance;
+};
+
+export function BalanceRow (
+  { code, name, coinCode, balance }: TProps
+) {
+  const { t } = useTranslation();
+
+  const [syncStatus, setSyncStatus] = useState<number>();
+
+  const unsubscribe = useRef<TUnsubscribe>();
+
+  const onSyncAddressesCount = (
+    _: string,
```

Comment:
this is the code string that is returned, reason is the follwoing line
(where code can change when changing accounts)

https://github.com/digitalbitbox/bitbox-wallet-app/blob/fe9bd5c92da6d8fa1bb7de539839e8db454a2ece/frontends/web/src/routes/account/account.tsx#L122

I think fine for this PR, we can improve that in the future, maybe use curry function.

Reply:
yup, would be nice to remove that, but I'd do it in a future PR